pfdp/css/fundraisers.css

```diff
@@ -0,0 +1,69 @@
+.cards {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  /* flex-grow: 1; */
+  /* grid-row-gap: 30px; */
+  /* grid-column-gap: 30px; */
+}
+
+.card {
+  margin: 15px 15px 15px 15px;
+  width: 363px;
+  height: 450px;
+  /* border: 1px solid black; */
+  position: relative;
+}
+
+.card-top {
+  min-height: 50%;
+  width: 100%;
+}
+
+.pill {
+  position: absolute;
+  height: 75px;
+  width: 75%;
+  background-color: var(--red);
+  border-radius: 0px 300px 300px 0px;
+  top: 40%;
+}
+.pill-text {
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 29px;
+  display: flex;
+  align-items: center;
+  color: #FFFFFF;
+  padding-top: 20px;
+  padding-left: 50px;
+}
+
+.card-bottom {
+  height: 30%;
+  max-width: 100%;
+  padding: 15% 5% 10% 5%;
+  background-color: var(--pastel);
+}
+
+.card-text {
+font-family: Montserrat;
+font-style: normal;
+font-weight: normal;
+font-size: 16px;
+line-height: 20px;
+/* display: flex; */
+align-items: center;
+}
+
+@media screen and (max-width: 500px) {
+  .cards {
+    grid-template-columns: 1fr;
+  }
+  .card {
+    width: 273px;
+    height: 338px;
```

Comment:
min-height here (same, see previous comment)

Reply:
changed **height** to **min-height**